getting-started/setup-building.rst

```diff
@@ -81,13 +81,19 @@ You will only need to execute these steps once:
       $ cd cpython
       $ git remote add upstream git@github.com:python/cpython.git
 
-7. Verify that your setup is correct::
+7. Configure ``git`` to pull main from the ``upstream`` remote::
+
+      $ git config --local branch.main.remote upstream
+
+8. Verify that your setup is correct::
 
       $ git remote -v
       origin  git@github.com:<your-username>/cpython.git (fetch)
       origin  git@github.com:<your-username>/cpython.git (push)
       upstream        git@github.com:python/cpython.git (fetch)
       upstream        git@github.com:python/cpython.git (push)
+      $ git config branch.main.remote
+      upstream
```

Comment:
This could be added to the Git bootcamp too:
https://devguide.python.org/getting-started/git-boot-camp/index.html#cloning-a-forked-cpython-repository

Reply:
Probably we should figure out a way to consolidate these two sets of instructions. Describe the steps in one place and link to them from another. My instinct is to provide the instructions in the git boot camp and link them from here. I'll probably do that in a prerequisite change prior to this one to avoid having to keep the two sets of instructions in sync. How does that sound?

Reply:
Having all the commands here is convenient, since people doing a setup don't have to follow links back and forth from this page to the GitHub bootcamp page.  On the other hand, the bootcamp entries are also convenient on their own [^1].

I think the best approach would be to:
* list all upstream-related commands in a single step and code block[^2], with short inline comments (about what the commands do)
* have the same commands in the bootcamp, but with more in-depth discussions (e.g. why they are necessary)
* from here, link to the bootcamp so that people that want more details can find them there
* add a couple of comments about the duplicated commands as a remainder for people updating either page

[^1]: I regular revisit the entry about setting up a remote, since I do it for every new repo I clone, but not often enough that I remember the exact synta
[^2]: this includes steps 6-9.  Step 5 can be separated.

Reply:
I'll work on this today.

Reply:
Done in 769a334. I've opted not to implement the "best approach" above for now, though I agree that would be a nice improvement.